src/erma.erl

```diff
@@ -77,7 +77,7 @@ resolve_placeholders(Query, Options) ->
     {Query2, lists:reverse(Params)}.
 
 
--spec resolve_placeholders(sql_query(), list(), erma_options()) -> {sql_query(), list()}.
+-spec resolve_placeholders(term(), list(), erma_options()) -> {sql_query(), list()}.
```

Comment:
I am pretty sure it should be `sql_query()` type. May be I am wrong, and there are calls to this function with argument other than `sql_query()`. Perhaps we should fix those calls, not the function spec.

Reply:
Yes you are right. If we take a look in `master` [erma/src/erma.erl#L93](https://github.com/yzh44yzh/erma/blob/master/src/erma.erl#L93) - we can see that function is expected a `list` - this because of [erma/src/erma.erl#L90](https://github.com/yzh44yzh/erma/blob/master/src/erma.erl#L90) recursive call. So, based on type [sql_query()](https://github.com/yzh44yzh/erma/blob/master/include/erma.hrl#L76) expected `tuple` only.